backend/spec/requests/memos_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+RSpec.describe 'Memos' do
+  describe 'MemoAPI' do
```

Comment:
describeでは、対象となるエンドポイントを表すようにしましょう〜！
Rspecはテストとしてだけでなく、仕様書としても意味を持つので！

Ref: https://paytner.hatenablog.com/entry/2023/04/04/190046

```suggestion
  describe 'POST /memos' do
```

Reply:
承知しました！
MemoAPIだと定義も広すぎますもんね・・・